ros/src/computing/perception/detection/lidar_detector/packages/lidar_point_pillars/package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package>
+    <name>lidar_point_pillars</name>
+    <version>1.9.1</version>
+    <description>lidar_point_pillars</description>
+
+    <maintainer email="cirpue49@yahoo.co.jp">Kosuke Murakami</maintainer>
+    <license>BSD</license>
+
+    <buildtool_depend>catkin</buildtool_depend>
+
+    <build_depend>roscpp</build_depend>
+    <build_depend>pcl_ros</build_depend>
+    <build_depend>cuda-toolkit-10-0</build_depend>
+    <build_depend>autoware_msgs</build_depend>
+
+
+    <run_depend>roscpp</run_depend>
+    <run_depend>pcl_ros</run_depend>
+    <run_depend>cuda-toolkit-10-0</run_depend>
```

Comment:
You might try `nvidia-cuda-dev`
https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml#L4109
However, it's not sure which version will be installed by this package. 

In my opinion, it might be better to remove this dependency. Let `FindCUDA`, and your CMake script to decide whether or not to compile the package.

Reply:
Thank you for the advice!